extensions/adobe/experience/journeyOrchestration/stepEvents/journeyStepEventActionExecutionFieldsMixin.schema.json

```diff
@@ -123,6 +123,73 @@
               "type": "integer",
               "meta:titleId": "journeyStepEventActionExecutionFields##https://ns.adobe.com/experience/journeyOrchestration/actionSchedulerCount##title##88741",
               "meta:descriptionId": "journeyStepEventActionExecutionFields##https://ns.adobe.com/experience/journeyOrchestration/actionSchedulerCount##description##46891"
+            },
+
+            "https://ns.adobe.com/experience/journeyOrchestration/messageID": {
+                "title": "MessageID",
+                "description": "Message ID referenced in inline message action of a journey",
+                "type": "string"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/publishedMessageID": {
+                "title": "PublishedMessageID",
+                "description": "Published message ID in inline message action of a journey",
+                "type": "string"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/interactionType": {
+                "title": "InteractionType",
+                "description": "Indicate the interaction type, either 'marketing' or 'transactional'. Remark: this will be relevant for message and custom actions",
+                "type": "string"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/surfaceID": {
+                "title": "SurfaceID",
+                "description": "Surface ID",
+                "type": "string"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/channel": {
+                "title": "Channel",
+                "description": "Channel Name",
+                "type": "string"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/isSendTimeOptimized": {
+                "title": "IsSendTimeOptimized",
+                "description": "Indicate if the message execution has been delayed by applying a send time optimization",
+                "type": "boolean"
+            },
+
+            "https://ns.adobe.com/experience/journeyOrchestration/actionGroupID": {
+                "title": "ActionGroupID",
+                "description": "Action Group ID",
+                "type": "string"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/actionGroupName": {
+                "title": "ActionGroupName",
+                "description": "Action Group Name",
```

Comment:
Can we add some examples in the description ?

Reply:
As discussed with Stephane, ActionGroupName will be a user-created value on UI and hence I think we should be good without example here. User will know the action group name created by them